tictactoe.py

```diff
@@ -28,7 +28,16 @@ def humans_move(game_board):
 
 def computers_move(game_board):
     """Update the board with the computer player's move if an empty cell exists"""
-    pass
+    num=len(game_board)
+    for i in range(num):
+        if game_board[i]!='E':
+            pass
+        else:
```

Comment:
Since the if-branch does nothing, you can do
```python
if game_board[i] == 'E':
    start = ...
    ...
    break
```
and omit an `else` branch.